src/MemoryManager.php

```diff
@@ -336,7 +346,7 @@ private function commit()
         $id = $container->getId();
 
         // Calculate new object size and increase used memory size by this value
-        $this->memorySize += ($this->sizes[$id] = strlen($container->getRef()));
+        $this->memorySize += $this->sizes[$id] = strlen($container->getRef());
```

Comment:
This change in brackets seems very confusing - not sure if operation order changed.